vipps-recurring-api.md

```diff
@@ -56,34 +56,46 @@ Agreements can be initiated with or i without an initial charge.
 
 **Intervals**\
 Intervals are defined with a interval type `YEAR`, `MONTH`, `WEEK`, or `DAY` and frequency as a count.\
+
 Example for a bi-weekly subscription:
 ```json
 "interval": "WEEK",
 "intervalCount": 2,
 ```
+
 Example for a quarterly subscription
 ```json
 "interval": "MONTH",
 "intervalCount": 3,
 ```
 
+Example for a yearly subscription
+```json
+"interval": "YEAR",
+"intervalCount": 1,
+```
+
 #### Initial charge
-Initial charge will be performed if the `initialcharge` is provided when creating an agreement. The `amount` has to correspond to the `price` of the agreement.
+Initial charge will be performed if the `initialcharge` is provided when
+creating an agreement. The `amount` has to correspond to the `price` of the agreement.
 
 ```json
 "initialCharge": {
-  "amount": 135,
+  "amount": 19900,
   "currency": "NOK",
-  "description": "Payment for September"
+  "description": "Premier League subscription: September"
```

Comment:
We do this concatenation of charge description and productName in the code now, this should just read as `September` now.